testing/e2e/clients/aks.go

```diff
@@ -38,6 +39,13 @@ type aks struct {
 	options                             map[string]struct{}
 }
 
+type ServicePrincipal struct {
+	ApplicationObjectID          string
```

Comment:
readonly for fields we dont add

Reply:
can u add the comments in a per field basis so that they show up in IDEs when accessed/when ppl hover over them